build.gradle

```diff
@@ -799,7 +800,7 @@ subprojects {
     skipConfigurations = [ "zinc" ]
   }
 
-  if (project.name in spotlessApplyModules) {
+  if (JavaVersion.current().isJava11Compatible() && (project.name in spotlessApplyModules)) {
     apply plugin: 'com.diffplug.spotless'
```

Comment:
another idea: Could we revers the `spotlessApplyModules` to be the "exclude list"? For example:
```
def excludedSpotlessModules = ['core', 'client']
```
and then we remove module one by one if we apply the spotless rule. It is more clear then before, since we can "see" which module is not applied. WDYT?

Reply:
Yes, good idea.